docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -25,14 +25,9 @@
 //  - attribute: :endpoint-artifact-name-single-sentence-start:
 //      The name of the artifact in singular form with first letter capitalized
 //      Example: Trusted application
-//
-//  DOCUMENT IDS:
-//  The reusable content in this file references the following ids which must exist in the document:
-//
-//  - [#create-item-api]
-//  - [#update-item-api]
-//  - [#exception-item-entry-schema]
-//
+//  - attribute: :endpoint-artifact-api-doc-name: 
```

Comment:
So this change actually makes it harder for someone to understand what attributes MUST be defined in their doc (outside of this `_exceptions-api-reusable-content.asciidoc` file) in order for the x-reference links to be valid/created.

I suggest that you either revert this (there is really only 3 document ids that must be defined outside of this reusable content in order for the x-references to work. Aside from those, every other document id is included in this file, and thus it is not necessary to document it here.

If you do decide to keep this change, then please document all of the required IDs that the document must have in order for the x-references used by this reusable content to work properly.

Reply:
Ok, but you still need to document the external dependencies (the two IDs) that need to exist in order for the reusable content to work, right? 

In order for the content of this file to be used, the document that `includes` content from here must have the following ids defined, which use the `:endpoint-artifact-api-doc-name:` attribute:

```
#create-{endpoint-artifact-api-doc-name}-item-api
#update-{endpoint-artifact-api-doc-name}-item-api
#exception-{endpoint-artifact-api-doc-name}-item-entry-schema
```

A little more background: most of the content here should really belong in a reusable file under the Exceptions API and we should just be able to reference (include) from there. I held off on doing that because I did not want to incur the delay to having endpoint artifact docs go into the 8.1 release. But improvement to the Exceptions docs is something that @yctercero mentioned might be needed at some point and if built taking this approach of having reusable content available, then we need to make sure that any "external" dependencies need to be stated.

Reply:
done 6ae5cd1d0d7fcd07b19c0b53a5342c737e88a1d1

Reply:
yeah moving the reusable content under `Exceptions API` makes sense.